include/continuous_action_response.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include <cstddef>
+#include <iterator>
+#include <vector>
+#include <string>
+
+namespace reinforcement_learning {
+  class api_status;
+
+  class continuous_action_response {
+  public:
+    continuous_action_response() = default;
+    ~continuous_action_response() = default;
+
+    float get_chosen_action() const;
+    void set_chosen_action(float action);
+
+    float get_chosen_action_pdf_value() const;
+    void set_chosen_action_pdf_value(float pdf_value);
```

Comment:
Don't continuous actions have mass and not density?

Reply:
Nope, continuous actions have density